backend/MoleculeOSSite/Models/LoginDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+
+namespace MoleculeOSSite.ModelsDTO
+{
+    public class LoginDTO
+    {
+        
+        public string UsernameOrEmail { get; set; }
+        
+        public string Password { get; set; }
```

Comment:
```suggestion
namespace MoleculeOSSite.Models.DTOs
{
    public class LoginDTO
    {
        public string Login { get; set; }
        public string Password { get; set; }
```
Oprócz tego dodaj w Models folder DTOs i tam wsadź LoginDTO.